api/provider/aws/test_aws/environment_scaler_test.go

```diff
@@ -0,0 +1,9 @@
+package test_aws
+
+import (
+	"testing"
+)
+
+func TestEnvironmentScale(t *testing.T) {
```

Comment:
You should add in the old tests back in verbatim - they are really extensive and do a good job of catching corner cases

Reply:
Thanks, I was wondering about that. I'll start working on that too.